packages/compodoc/src/executors/build/executor.ts

```diff
@@ -135,6 +135,10 @@ function toCompodocOptions(
     disableSearch: options.disableSearch,
     disableDependencies: options.disableDependencies,
 
+    coverageTest: options.disableCoverage ? 0 : options.coverageTest ?? 70,
```

Comment:
Is this check for `disableCoverage` necessary?
What happens if we always pass the defaulted `coverageTest` (`70`) and `disableCoverage` is `true`?